test/karma.conf.cjs

```diff
@@ -4,19 +4,27 @@ module.exports = function(config) {
   config.set({
 
     // Base path, that will be used to resolve files and exclude
-    basePath: '',
+    basePath: '../',
 
     // Testing frameworks
     frameworks: ['mocha', 'chai'],
 
     // List of files / patterns to load in the browser
     files: [
-      '../node_modules/eventemitter2/lib/eventemitter2.js',
-      '../node_modules/cbor-js/cbor.js',
-      '../src/util/cborTypedArrayTags.js',
-      '../build/roslib.js',
-      './require-shim.js',
-      '*.test.js'
+      // Make some file available to be fetched/imported, but not included in the HTML.
+      {pattern: 'build/roslib.esm.js', serve: true, included: false},
+      {pattern: 'test/util/shim/chai.js', serve: true, included: false},
+      {pattern: 'src/util/shim/xmldom.js', serve: true, included: false},
+      {pattern: 'src/util/shim/cbor.js', serve: true, included: false},
+      {pattern: 'src/util/cborTypedArrayTags.js', serve: true, included: false},
+
+      // This file maps some import specifiers to our module shims.
+      'test/util/write-importmap.js',
+
+      // Installs cbor-js in a global CBOR variable so that our shim/cbor.js shim picks it up.
+      'node_modules/cbor-js/cbor.js',
+      
+      {pattern: 'test/*.test.js', type: 'module'},
```

Comment:
`type: 'module'` makes this file operate as native browser `<script type="module"` , and therefore the passing tests are exercising the new `roslib.esm.js` output file in browser ESM.